### DISCUSSION TITLE
Cannot access copilot from VS Code Insiders

### DISCUSSION THREAD

**Discussion by rajatvijay** [community/community#53111] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I was using the technical preview of the Gtihub copilot but that has ended now. My organization has bought paid access to the GitHub copilot. It has also allowed me to use that access. My GitHub profile's setting also says that I have access to the copilot from my org but I am not able to use it from my vs code IDE.

## VS Code Logs

<img width="975" alt="Screenshot 2023-04-18 at 5 07 17 PM" src="https://user-images.githubusercontent.com/11032856/232765342-db677606-8aef-4d0f-b299-b3919530246b.png">

**Comment by Akash1134** [comment]
Hi @rajatvijay ,

Sorry that you're unable to use Copilot and getting an error 403.

A common cause of a 403 error is an attempt to authenticate to Copilot with a GitHub account that does not have a Copilot subscription. For example, if you have more than one GitHub account (one for work and a personal one), you might inadvertently be signed into the account without a Copilot subscription when trying to authenticate.

In VS Code, please use the accounts menu in the lower left-hand sidebar to log out of GitHub, and then log back in, making sure you're signing into the correct GitHub account.

If that doesn't resolve your issue, please reach out to our [Support](https://support.github.com/contact) team for account-specific help.

**Comment by rajatvijay** [comment]
@Akash1134 I have double-checked. I only have one Github account and that account has copilot access from the org. 

One thing to note here is: I have one more email listed under my account through which I have the org access on GitHub.

**Comment by rajatvijay** [comment, marked as answer]
This is resolved now. I had to make my company's email the primary and re-ask for access from the org admin.
